src/components/cards/EthCard.tsx

```diff
@@ -76,9 +75,9 @@ export const EthCard = () => {
   );
 
   const handleAssetPress = useCallback(() => {
-    if (!ethAsset) return;
+    if (ethAsset.native == null) return;
     navigate(Routes.EXPANDED_ASSET_SHEET_V2, {
-      asset: ethAsset,
+      asset: ethAsset as FormattedExternalAsset,
```

Comment:
Are you ok with this? I now see better why this change was made. For some reason ts wasn't running properly in my editor earlier.

Reply:
can fix it up later, this is fine for now